res/values/strings.xml

```diff
@@ -791,6 +791,8 @@
     <string name="qraccount_success_enter_name">Login successful—your e-mail address is %1$s\n\nIf you like, you can now enter a name and an profile image that will be displayed to people you write to.</string>
     <string name="qraccount_qr_code_cannot_be_used">The scanned QR code cannot be used to set up a new account.</string>
     <string name="qraccount_use_on_new_install">The scanned QR code is for setting up a new account. You can scan the QR code when setting up a new Delta Chat installation.</string>
+    <string name="qrlogin_ask_login">Login to \"%1$s\"?</string>
```

Comment:
"Login" is the noun/adjective and "Log in" the verb. this is also how we use that elsewhere.

Reply:
ah, that bit os not fixed, will push that directly on master before uploading the strings to transifex.